markdown.go

```diff
@@ -111,23 +111,27 @@ flag is the %s query parameter below:
 
 ::: tip
 Some arguments may belong only to the CLI but appear here too. These usually
-belong to client-side processing of input, particularly in the `+"`add`"+` command.
+belong to client-side processing of input, particularly in the `+"`add`"+`
+command.
 :::
 
 
 ## HTTP status codes
 
 Status codes used at the RPC layer are simple:
 
+- `+"`200`"+` - The request was processed or is being processed (streaming)
 - `+"`500`"+` - RPC endpoint returned an error
 - `+"`400`"+` - Malformed RPC, argument type error, etc
 - `+"`403`"+` - RPC call forbidden
 - `+"`404`"+` - RPC endpoint doesn't exist
 - `+"`405`"+` - HTTP Method Not Allowed
 
-In other words, `+"`500`"+` means that the function _does_ exist, it just
-failed internally for some reason. To know that reason, you have to look at
-the "application layer" error (usually returned with the body of the command).
+Status code `+"`500`"+` means that the function _does_ exist, it just failed
+internally for some reason. To know that reason, you have to look at the
+"application layer" error (usually returned with the body of the command). In
+the case of streaming endpoints, they always return 200 before starting to
+stream the response. Any errors are included as Trailer response headers.
```

Comment:
If we return an error before we start streaming, I think we may return a 500.

It's simpler to just say that 4xx are transport-level errors. For application level errors, you may either receive a 500 error code if the error is detected early, or a 200 response plus a trailing error if it's detected late.

Reply:
I want to avoid talking about transport layer vs application layer as that may not be understandable for some users. I gave it another try and tried to explain this case.